scripts/governance/sdk/actions.py

```diff
@@ -220,3 +278,17 @@ def query_proposal(proposal_id):
     req = {"proposal_id": proposal_id}
     resp = rpc_call('query.GetProposalByID', req)
     return resp["result"]
+
+
+def query_balance(address):
```

Comment:
Are these 2 funcs same?

Reply:
yea, will delete it... must be a copy paste mistake